<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HumanInfo</class>
 <widget class="QDialog" name="HumanInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>361</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Фамилия</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Отчество</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Адрес</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>День рождения</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Номер карты</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>UID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="name1"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="name2"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="name3"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="adress"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="birthday"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="cardname"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="uid"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
